.github/workflows/push-ci.yml

```diff
@@ -44,21 +44,25 @@ jobs:
         name: cryostat-amd64
       if: github.repository_owner == 'cryostatio'
     - name: Load cryostat-amd64 image
-      run: podman load -i cryostat-amd64.tar
+      run: |
+          podman load -i cryostat-amd64.tar
+          podman tag quay.io/cryostat/cryostat:latest $CRYOSTAT_IMG:$IMAGE_VERSION-linux-amd64
```

Comment:
That should be fine. It won't break anything for the release branches right? They'll just use the `latest` tag locally?

Reply:
https://github.com/cryostatio/cryostat/blob/20ed482b11c6d8f1a4b0a650175f59f574f1f757/.github/workflows/ci-build-image.yml#L78

Here is where the image tarball gets created, and it tags it as `quay.io/cryostat/cryostat`, so the `:latest` must be implied.

Reply:
Yea I think it should be fine, in the end what _should_ end up happening here is that future release branches will still get named like `quay.io/cryostat/cryostat:2.4.0`, but rather than being a single-arch concert image, that would be a manifest that contains `quay.io/cryostat/cryostat:2.4.0-linux-amd64` and ``quay.io/cryostat/cryostat:2.4.0-linux-arn64`. The `:latest` here doesn't get pushed, it's just what the podman registry on the runner instance thinks the particular image is called.